### PR TITLE
Custom path style

### PR DESCRIPTION
**Pull request by jeddi**
I added an option `customStyle` to relations.
In my use case i need to animate a `strokeDasharray`
I added a third example to demo

### CODE REVIEW

**Comment by jfo84** [comment]
The example isn't necessary. This would make more sense if you had a generic `arrow` prop under `style` and merged `stroke` and `strokeWidth` under this key

**Comment by jfo84** [comment]
Also please include the `yarn.lock` if you want this merged

**Comment by jeddi** [comment]
Do you mean something like this ?
``` jsx
        <ArcherElement
            id="elementA"
            relations={[
              {
                targetId: 'elementB',
                targetAnchor: 'left',
                sourceAnchor: 'right',
                style: {
                  arrow: {
                    strokeWidth: 1,
                    stroke: 'blue',
                    // others
                    opacity: 0.5,
                    strokeDasharray: 16px,
                  }
                },
              },
            ]}
          >
            <div style={boxStyle}>Element 2</div>
          </ArcherElement>
```

The `yarn.lock` is unchanged.

**Comment by jfo84** [comment]
@jeddi Yes, exactly. You are using an outdated version of yarn

**Comment by pierpo** [comment]
> Also please include the `yarn.lock` if you want this merged

I think you meant "please do not include", right?

**Comment by jeddi** [comment]
Requested changes is done.
I use the latest version of `yarn` (1.16.0)

File: yarn.lock [hunk 1]
```diff
@@ -5,14 +5,12 @@
 "@babel/code-frame@^7.0.0", "@babel/code-frame@^7.0.0-beta.35":
   version "7.0.0"
   resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.0.0.tgz#06e2ab19bdb535385559aabb5ba59729482800f8"
-  integrity sha512-OfC2uemaknXr87bdLUkWog7nYuliM9Ij5HUcajsVcMCpQrcLmtxRbVFTIqmcSkSeYRBFBRxs2FiUqFJDLdiebA==
```
Comment (pierpo) [inline yarn.lock]: Can you remove your changes in the `yarn.lock`? They are completely unrelated to your feature 😉

Comment (jeddi) [reply yarn.lock]: i'ts done !
Sorry, i did not notice that the file was changed (`git diff` does not show changes in this file !)

**Comment by pierpo** [comment]
Sorry about the very long delay, I still need to double check everything, update the types and the documentation...

**Comment by pierpo** [comment]
@jeddi Why did you close this? You don't need this anymore?
It feels like it was relevant to keep it 😉

**Pull request closed without merging**